include/z64actor.h

```diff
@@ -8,7 +8,6 @@
 
 #define ACTOR_NUMBER_MAX 200
 #define INVISIBLE_ACTOR_MAX 20
-#define AM_FIELD_SIZE 0x27A0
 #define MASS_IMMOVABLE 0xFF // Cannot be pushed by OC colliders
```

Comment:
These aren't really the same sort of thing, so there should probably at least be a blank line here.